### DISCUSSION TITLE
[Question] How to hide or exclude some info from the recording?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [asciinema/discussions#240] [Q&A] [answered]
Hello, I just tested asciinema

It displays my user name and some of my information that I display for myself in my shell.
How would I go about hiding those?

**Comment by ku1ik** [comment]
You mean your shell's prompt? E.g. `user@hostname $` or similar? asciinema records everything that is printed in your terminal.

But you can check for presence of `ASCIINEMA_REC` env variable, which is set when recording. You can add a conditional in your shell's config file to change prompt template (hide sensitive info) when this env var is set.

**Reply by BarbzYHOOL** [reply]
so i'd have to script something to hide my own information?

**Reply by ku1ik** [reply, marked as answer]
If you decide to record your terminal session with asciinema and you don't want something to be part of the recording then you need to hide it, yes. It's the same with other terminal recording programs like `script`, `ttyrec` or `termrec`, nothing special here.

**Reply by BarbzYHOOL** [reply]
what a hassle :(

**Reply by ku1ik** [reply]
Feel free to use another tool, one which suits you better.

**Reply by BarbzYHOOL** [reply]
you said there's none
